editor/components/inserter/menu.js

```diff
@@ -229,9 +228,9 @@ export class InserterMenu extends Component {
 		// If the Saved tab is selected and we have no results, display a friendly message
 		if ( 'saved' === tab && itemsForTab.length === 0 ) {
 			return (
-				<p className="editor-inserter__no-tab-content-message">
-					{ __( 'No saved blocks.' ) }
-				</p>
+				<NoBlocks>
+					No saved blocks
```

Comment:
It should remain wrapped with translation function:
```js
{ __( 'No saved blocks' ) }
```

Reply:
My bad 😓